app/views/state_file/questions/nj_medical_expenses/edit.html.erb

```diff
@@ -0,0 +1,46 @@
+<%
+  title = t(".title", filing_year: @filing_year)
+  content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= t(".title") %></h1>
+  <ul class="list--bulleted">
+    <li><%= t(".includes_whom") %></li>
+    <li><%= t(".percent_income") %></li>
+    <li><%= t(".no_reimbursed") %></li>
```

Comment:
[pebble] You can put this bullet list in a single content item, which would probably be easier to read and edit than tracking them as independent content items. See `terms_and_conditions.edit.ny_text_html` as an example